apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java

```diff
@@ -418,7 +444,8 @@ public void setOut(final File file) {
                 DefaultLog.getInstance().warn("Unable to delete file: " + file);
             }
         }
-        if (!file.getParentFile().mkdirs()) {
+        File parent = file.getParentFile();
+        if (!parent.mkdirs() && !parent.isDirectory()) {
```

Comment:
When reading this line I stumbled if it can be simplified ....

NOT(a) and NOT(b) == a OR b ?!

But I'm not sure if it is easier to read as 
`!parent.mkdirs() && parent.isFile()` .... WDYT?